src/app/detail/[task_id]/[user_type]/page.tsx

```diff
@@ -57,6 +57,7 @@ async function getSignedUrlUpload(token: string, fileName: string) {
 }
 
 async function getSignedUrlFile(token: string, filePath: string) {
+  'use server'
```

Comment:
what is the purpose of using `use server` here? This is already a server component and also this is not a correct pattern. Please follow this - https://github.com/copilot-platforms/tasks-app/pull/286#discussion_r1760541847